src/stacky_ffi.erl

```diff
@@ -2,6 +2,14 @@
 
 -export([stacky_erlang_stack_trace/0]).
 
+enumerate_reversed(List) ->
+    I = length(List),
+    enumerate_reversed(I, 1, List).
+
+enumerate_reversed(I, S, List) ->
+    {List1, _ } = lists:mapfoldl(fun(T, Acc) -> {{Acc, T}, Acc-S} end, I, List),
+    List1.
```

Comment:
Implementation based on what's illustrated [in the docs](git@github.com:korokd/stacky.git)

Reply:
in which docs?

Reply:
and... would you add a note to the changelog?

Reply:
> in which docs?

Oops, had an issue of malformatted markdown, I updated my comment to correctly link to it.

Reply:
> and... would you add a note to the changelog?

Sure